docs/quickstart/azure.md

```diff
@@ -95,32 +86,27 @@ Press `enter` to select `Develop in Linux on Azure` template. This will return t
 
 To get started using the Azure template, install the Azure CLI by following the instructions here. Run `az login` and follow the instructions to configure the Azure command line. 
 
-For this example, Coder was run as a system level process, which creates the system user `coder` for handling processes. The coder user will require access to the Azure credentials to initialize the template.
+For this example, Coder was run as a system level process, which creates the system user `coder` for handling processes. The Coder user will require access to the Azure credentials to initialize the template.
+
 Run the following commands to copy the Azure credentials and give the `coder` user access to them:
 
-```bash
+```sh
 sudo cp -r ~/.azure /home/coder/.azure 
 sudo chown -R coder:coder /home/coder/.azure/
 ```
 
+Navigate to the `/azure-linux` folder where you created your template and run the following command to put the template on your Coder instance. 
 
-Navigate to the `/azure-linux` folder where you created your template and run the following command to put the template on your coder instance. 
-
-```
+```sh
 coder templates create
 ```
 
-
 Congrats! You can now navigate to your Coder dashboard and use this Linux on Azure template to create a new workspace!
 
+## Troubleshooting
 
-## Debugging Help
-
-If you’re not sure if Coder is running correctly as a system service, use the command `systemctl status coder` to show the status of the coder process.
-
-
-To stop running coder as a system process, run the following command: `systemctl stop coder`. 
-
-When you run ` journalctl -u coder.service -b`, there’s a chance that it may not show any logs, preventing you from seeing the url for the running coder instance. Use the command `systemctl stop coder` to stop the coder system process, and run the command to start Coder as a system process again. You can also use the command `systemctl restart coder` to quickly restart the process. 
+If you’re not sure if Coder is running correctly as a system service, use the command `systemctl status coder` to show the status of the Coder process.
 
+To stop running Coder as a system process, run the following command: `systemctl stop coder`. 
 
+When you run ` journalctl -u coder.service -b`, there’s a chance that it may not show any logs, preventing you from seeing the url for the running Coder instance. Use the command `systemctl stop coder` to stop the Coder system process, and run the command to start Coder as a system process again. You can also use the command `systemctl restart coder` to quickly restart the process. 
```

Comment:
Why not suggest they use `journalctl --grep` instead? I see your comment in Google Docs: "this is for just seeing if its running?", which I don't understand. This paragraph assumes the service is already running since you're recommending they stop it.

Reply:
specific command was at the instruction of ben so that was why i included it as the choice instruction